THC: Cleanup dead code

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #64442
* __->__ #64441

Differential Revision: [D30735342](https://our.internmc.facebook.com/intern/diff/D30735342)

peterbell10 committed:
THC: Cleanup dead code

[ghstack-poisoned]

ngimel commented:
@ngimel has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30735342).

peterbell10 committed:
Update on "THC: Cleanup dead code"


Differential Revision: [D30735342](https://our.internmc.facebook.com/intern/diff/D30735342)

[ghstack-poisoned]

ngimel commented:
@ngimel has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30735342).
